androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -259,6 +261,16 @@ protected Uri getLaunchingUrl() {
     }
 
     protected TwaLauncher.FallbackStrategy getFallbackStrategy() {
+        if (FALLBACK_TYPE_WEBVIEW.equalsIgnoreCase(mMetadata.fallbackStrategyType)) {
```

Comment:
Could we make this a member of TwaLauncher like CCT_FALLBACK_STRATEGY (eg, WEBVIEW_FALLBACK_STRATEGY)?

Reply:
done